src/activity/index.html

```diff
@@ -21,6 +21,7 @@
   <body>
     <header></header>
     <body>
```

Comment:
Change this nested body element to a main element. Like the one I wrote earlier. Thank you!